libraries/Microsoft.Bot.Schema/ActivityEx.cs

```diff
@@ -407,42 +407,38 @@ public Mention[] GetMentions()
         /// <summary>
         /// Gets the channel data for this activity as a strongly-typed object.
         /// </summary>
-        /// <typeparam name="TypeT">The type of the object to return.</typeparam>
+        /// <typeparam name="TType">The type of the object to return.</typeparam>
```

Comment:
I think this change can be included. I know this might involve some conflicts. But the TType thing is just an identifier. Right?

Reply:
Yes, I included it here because it was part of the same file, but I can move it to the other PR.

Reply:
Done, thanks!